actions/admin/record.go

```diff
@@ -11,8 +11,16 @@ import (
 )
 
 // transactionRecord will save and complete a transaction directly, without any checks
+// Record transactions godoc
+// @Summary      Record transactions
+// @Description  Record transactions
+// @Tags		 Admin
+// @Produce      json
+// @Param hex query string true "Transaction hex"
+// @Success      200
```

Comment:
Not your fault again. But this is a "Record"("Create") function. But success code is 200(OK). Shouldn't it be 201(Created)?